### ISSUE TITLE
PM Workflow to check if assignee is blocked or has abandoned issue

### ISSUE THREAD

**Issue by mekarpeles** [internetarchive/openlibrary#9691]
### Problem

Leads assign issue that sometimes get abandoned (e.g. no linked PR after 2 weeks, or linked PR but issue/PR is blocked)

### Impact

Blocked or abandoned issues prevent others from becoming assigned to the issue

### Solution

Probably not to auto-assign but rather, similar to comment bot, flag these issues with “Needs: Review Assignee” label so the team may evaluate and review + we can show in the slack digest bot.

### Idea

Followup idea if necessary: we could also post a message to the assignee and say, "Hi `@person` are you still working on this issue or is there anything you're blocked on? We're here to help"

### Stakeholders

<hr>

### Instructions for Contributors

- Please [run these commands](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#working-on-your-branch) to ensure your repository is up to date **before** [creating a new branch](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#making-changes-and-creating-a-pull-request) to work on this issue and **each time after** pushing code to Github, because the pre-commit bot may add commits to your PRs upstream.

**Comment by mekarpeles** [comment]
Close as duplicate of #9657
